cartridges/plugin_gtm/cartridge/templates/default/gtm/gtmScript.isml

```diff
@@ -1,7 +1,82 @@
+<iscomment>
+The "gtm-metadata" div and the additional logic below was added to resolve a specific issue
+occurring on mobile platforms only.
+
+What will happen is, if the customer purchases an item, sees the order confirmation page and then
+leaves the order confirmation open in their browser; e.g. Safari, Chrome both demonstrated this,
+then after the app connection times out or if the customer "swipe closes" the app, as you can on iOS,
+when the app reloads, while it would not initiate a new call to the server, it does reload what it has
+in its local cache, which includes re-executing javascript.
+
+This meant that the call to "push" the data layer would happen again. This was noticed in Google
+Analytics where a single transaction for $50.00 would have a value greater than that; $150, $250, maybe more,
+depending on how many times it had been "re-pushed" to GA.
+As one can imagine, given a client with a larger customer base, this could greatly throw off the numbers in
+GA compared to what is in SFCC or the customer's backend OMS/ERP.
+
+It is worth noting that this solution uses localStorage. Local Storage is the same across the entire browser.
+So, cannot simply store true and false for "the order confirmation data layer has been sent", because then
+subsequent orders would never have their data layer pushed. Orders sent are tracked by storing the orderId.
+</iscomment>
```

Comment:
Can you add the comment to the PR not the code.

Reply:
Info for this PR

The logic change in this branch was added to resolve a specific issue occurring on mobile platforms only.

What will happen is, if the customer purchases an item, sees the order confirmation page and then
leaves the order confirmation open in their browser; e.g. Safari, Chrome both demonstrated this,
then after the app connection times out or if the customer "swipe closes" the app, as you can on iOS,
when the app reloads, while it would not initiate a new call to the server, it does reload what it has
in its local cache, which includes re-executing javascript.

This meant that the call to "push" the data layer would happen again. This was noticed in Google
Analytics where a single transaction for $50.00 would have a value greater than that; $150, $250, maybe more,
depending on how many times it had been "re-pushed" to GA.
As one can imagine, given a client with a larger customer base, this could greatly throw off the numbers in
GA compared to what is in SFCC or the customer's backend OMS/ERP.

It is worth noting that this solution uses localStorage. Local Storage is the same across the entire browser.
So, cannot simply store true and false for "the order confirmation data layer has been sent", because then
subsequent orders would never have their data layer pushed. Orders sent are tracked by storing the orderId.